cloudtrail-s3/main.tf

```diff
@@ -1,16 +1,19 @@
+locals {
+  bucket_name = "cloudtrail-${var.project}-${var.environment}"
```

Comment:
this will force existing setups that don't follow this naming structure to recreate the bucket. Could you add a disclaimer / migration path in the release notes?

Reply:
yep indeed